src/blocks/donate/streamlined/index.ts

```diff
@@ -108,6 +108,14 @@ export const processStreamlinedElements = ( parentElement = document ) =>
 				}
 			}
 			const formValues = utils.getDonationFormValues( formElement );
+			const promptOrigin = formElement.closest( '.newspack-popup' );
```

Comment:
To work with both inline & overlay prompts, it should target `amp-layout`.

```suggestion
			const promptOrigin = formElement.closest( 'amp-layout.newspack-popup' );
```

In an overlay prompt, there's a child of `amp-layout` that also gets `.newspack-popup` class. Which should not be happening, I've added https://github.com/Automattic/newspack-popups/issues/945